downstream/titles/release-notes/async/aap-25-5-3-dec.adoc

```diff
@@ -8,33 +8,33 @@ The following enhancements and bug fixes have been implemented in this release o
 
 === {PlatformNameShort}
 
-* {LightspeedShortName} has been updated to 2.5.241127.
+* {LightspeedShortName} has been updated to 2.5.241127.(AAP-35307)
 
-* `redhat.insights` Ansible collection has been updated to 1.3.0. 
+* `redhat.insights` Ansible collection has been updated to 1.3.0.(AAP-35161)
 
-* `ansible.eda` collection has been updated to 2.2.0 in {ExecEnvShort} and decision environment images. 
+* `ansible.eda` collection has been updated to 2.2.0 in {ExecEnvShort} and decision environment images.(AAP-3398)
 
 === {OperatorPlatformNameShort}
 
-* With this update, you can set PostgreSQL SSL/TLS mode to `verify-full` or `verify-ca` with the proper `sslrootcert` configuration in the {HubName} Operator.
+* With this update, you can set PostgreSQL SSL/TLS mode to `verify-full` or `verify-ca` with the proper `sslrootcert` configuration in the {HubName} Operator.(AAP-35368)
 
 === Container-based {PlatformNameShort}
 
-* With this update, `ID` and `Image` fields from a container image are used instead of `Digest` and `ImageDigest` to trigger a container update. 
+* With this update, `ID` and `Image` fields from a container image are used instead of `Digest` and `ImageDigest` to trigger a container update.(AAP-36575)
 
-* With this update, you can now update the registry URL value in {EDAName} credentials. 
+* With this update, you can now update the registry URL value in {EDAName} credentials.(AAP-35085)
 
-* With this update, the `kernel.keys.maxkeys` and `kernel.keys.maxbytes` settings are increased on systems with large memory configuration. 
+* With this update, the `kernel.keys.maxkeys` and `kernel.keys.maxbytes` settings are increased on systems with large memory configuration.(AAP-34019)
 
-* Added `ansible_connection=local` to the `inventory-growth file` and clarified its usage. 
+* Added `ansible_connection=local` to the `inventory-growth file` and clarified its usage. (AAP-34016)
```

Comment:
For consistency, there should either be a space or no space between the statement and the Jira ticket. Up to this point, there have been no spaces.